### PR TITLE
Show that we "intend" to add all files

### PR DESCRIPTION
**Pull request by Nyholm**
https://stackoverflow.com/questions/855767/can-i-use-git-diff-on-untracked-files

### CODE REVIEW

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
           git status
           echo "::endgroup::"
 
+          git add -N .
```
Comment (jderusse) [inline .github/workflows/website.yml]: should we use same for `watch.yaml`?

**Comment by Nyholm** [comment]
Woho!. It worked!

https://async-aws.com/

File: .github/workflows/website.yml (hunk 1 above)
Comment (Nyholm) [reply .github/workflows/website.yml]: Sure, why not? 

But it is unlikely that the watch command generates a new file.

Comment (jderusse) [reply .github/workflows/website.yml]: Replace string by Enum class for instance..
But yeah, in that case, at least one file would probably changed too

**Pull request merged**